docs/user-guide/metadata.md

```diff
@@ -110,19 +112,36 @@ $ meta get foo
 Example repo: <https://github.com/screwdriver-cd-test/workflow-metadata-example>
 
 Notes:
+
 - If `foo` is not set and you try to `meta get foo`, it will return a string with value `null` by default.
 
+### Restart case
+
+The metadata from the parent event will be accessible for use in the restarted build as well as in any downstream builds.
+
+Example: `build1` -> `build2` -> `build3` are builds in event1.
+
+Upon completion of `build3`, the metadata from `build1`, `build2`, and `build3` becomes available in `event1`. In other words, the metadata from each completed build is merged into the event in which the build is currently running.
+
+When `build1` is restarted, it will have access to metadata from `event1`. This metadata from `build1` will then be accessible to downstream builds, allowing `build2` and `build3` to also receive the metadata from `event1`.
+
+_Caveat_
+
+If a build is restarted from a parent event containing only one build that has never been run (e.g., a virtual job or frozen build), the restarted build will not inherit any metadata because the parent event’s metadata is empty.
```

Comment:
```suggestion
#### Caveat

If a build is restarted from a parent event containing only one build that has never been run (e.g., a virtual job or frozen build), the restarted build will not inherit any metadata because the parent event’s metadata is empty.
```